ios/Podfile

```diff
@@ -1,38 +1,57 @@
 # Uncomment the next line to define a global platform for your project
-platform :ios, '9.0'
+platform :ios, '10.0'
 
 target 'LNDR' do
   # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
   # use_frameworks!
 
   # Pods for LNDR
-  pod 'UrbanAirship-iOS-SDK', '~>9.0'
-  pod 'PayPal-iOS-SDK'
-  pod 'Firebase/Core', '~> 5.11.0'
-  pod 'Firebase/DynamicLinks', '~> 5.11.0'
+
+  pod 'Firebase'
+  pod 'Firebase/Core'
+  pod 'Firebase/DynamicLinks'
   pod 'Firebase/Analytics'
 
-  # target 'LNDR-tvOSTests' do
-  #   inherit! :search_paths
-    # Pods for testing
-  # end
+  pod 'UrbanAirship-iOS-SDK'
 
-  # target 'LNDRTests' do
-    # inherit! :search_paths
-    # Pods for testing
-  # end
+  pod 'PayPal-iOS-SDK'
 
-end
+  # pod 'React', :path => '../node_modules/react-native', :subspecs => [
+  #   'Core',
+  #   'CxxBridge', # Include this for RN >= 0.47
+  #   'DevSupport', # Include this to enable In-App Devmenu if RN >= 0.43
+  #   'RCTText',
+  #   'RCTNetwork',
+  #   'RCTWebSocket', # Needed for debugging
+  #   'RCTAnimation', # Needed for FlatList and animations running on native UI thread
+  #   'RCTImage'
+  #   # Add any other subspecs you want to use in your project
+  # ]
 
-# target 'LNDR-tvOS' do
-  # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
-  # use_frameworks!
+  # pod 'DoubleConversion', :podspec => '../node_modules/react-native/third-party-podspecs/DoubleConversion.podspec'
+  # pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
+  # pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
+
+  # pod 'yoga', :path => '../node_modules/react-native/ReactCommon/yoga'
+
+  # pod 'react-native-blur', :path => '../node_modules/react-native-blur'
 
-  # Pods for LNDR-tvOS
+  # pod 'react-native-fetch-blob', :path => '../node_modules/react-native-fetch-blob'
 
-  # target 'LNDR-tvOSTests' do
-  #   inherit! :search_paths
-    # Pods for testing
-  # end
+  # pod 'react-native-image-picker', :path => '../node_modules/react-native-image-picker'
 
-# end
+  # pod 'react-native-image-resizer', :path => '../node_modules/react-native-image-resizer'
+
+  # pod 'react-native-randombytes', :path => '../node_modules/react-native-randombytes'
+
+  # pod 'react-native-splash-screen', :path => '../node_modules/react-native-splash-screen'
+
+  # pod 'react-native-sqlite-storage', :path => '../node_modules/react-native-sqlite-storage'
+
+  # pod 'TouchID', :path => '../node_modules/react-native-touch-id'
+  
+  # pod 'RNVectorIcons', :path => '../node_modules/react-native-vector-icons'
+
+  # pod 'RNGestureHandler', :path => '../node_modules/react-native-gesture-handler'
+
+end
```

Comment:
Lots of commented-out lines here. Best to remove code rather than comment it out (leaving it in is a bit confusing, because it makes me second-guess whether there's something actually reading that code, like a post-processor of some kind).